core/src/main/scala/com/codacy/analysis/core/utils/HttpHelper.scala

```diff
@@ -15,9 +15,17 @@ class HttpHelper(apiUrl: String, extraHeaders: Map[String, String]) {
     val headers: Map[String, String] = Map("Content-Type" -> "application/json") ++ extraHeaders
 
     val response: HttpResponse[String] =
-      Http(s"$remoteUrl$endpoint").headers(headers).timeout(connectionTimeoutMs, readTimeoutMs).asString
+      if (allowUnsafeSSL)
+        Http(s"$remoteUrl$endpoint")
+          .headers(headers)
+          .timeout(connectionTimeoutMs, readTimeoutMs)
+          .options(HttpOptions.allowUnsafeSSL)
+          .asString
+      else
+        Http(s"$remoteUrl$endpoint").headers(headers).timeout(connectionTimeoutMs, readTimeoutMs).asString
```

Comment:
Here we can avoid this duplication.
`.options()` takes a `Seq[HttpOptions.HttpOption]` as parameter, so you can pass the option based on the boolean flag.

```suggestion
    val httpOptions = if (allowUnsafeSSL) Seq(HttpOptions.allowUnsafeSSL) else Seq.empty
    val response = Http(s"$remoteUrl$endpoint")
      .headers(headers)
      .timeout(connectionTimeoutMs, readTimeoutMs)
      .options(httpOptions)
      .asString
```